lib/formatters.js

```diff
@@ -134,14 +134,32 @@ function colorize(color, str) {
  * @deprecated
  */
 function formatTrace(level, context, message, args, err) {
-  args.unshift(
-      templateTrace + message,
-      (new Date()).toISOString(),
-      level,
-      context.corr || notAvailable,
-      context.trans || notAvailable,
-      context.op || notAvailable
-  );
+
+  var recontext = {
+    time: (new Date()).toISOString(),
+    lvl: level,
+    corr: context.corr || notAvailable,
+    trans: context.trans || notAvailable,
+    op: context.op || notAvailable
```

Comment:
Just as a way for everyone to learn, starting by myself :), I meant:
```javascript
var recontext = {
  time: (new Date()).toISOString(),
  lvl: level,
  corr: notAvailable,
  trans: notAvailable,
  op: notAvailable
};
```
This way, these fields (`corr`, `trans`, and `op`) would always be included in the final formatted message, wouldn't they? Or am I missing anything? :)
Anyhow, since the PR is landed, nothing more to say... :)